client.cpp

```diff
@@ -938,8 +982,8 @@ void run_stats::save_csv_one_sec_cluster(FILE *f,
                                          unsigned long int& total_set_ops,
                                          unsigned long int& total_wait_ops) {
     fprintf(f, "Per-Second Benchmark Data\n");
-    fprintf(f, "Second,SET Requests,SET Average Latency,SET Total Bytes,SET Moved,"
-               "GET Requests,GET Average Latency,GET Total Bytes,GET Misses,GET Hits,GET Moved,"
+    fprintf(f, "Second,SET Requests,SET Average Latency,SET Total Bytes,SET Moved, SET Ask,"
```

Comment:
Not sure about breaking output format just for the sake of ASK stats.  I'd either make this optional or at least push it to the end of the CSV to minimize risk of breaking something.